debezium-testing/debezium-testing-system/pom.xml

```diff
@@ -44,14 +44,16 @@
     <ocp.project.db2>debezium-db2</ocp.project.db2>
 
     <ocp.image.mysql>${docker.image.mysql}</ocp.image.mysql>
-    <ocp.image.postgresql>${docker.image.postgresql}</ocp.image.postgresql>
+    <ocp.image.postgresql>quay.io/debezium/example-postgres-ocp:latest</ocp.image.postgresql>
```

Comment:
Is this intentional? Other images seems still to be configurable

Reply:
This one is configurable as well. However for OCP a default docker image is not usable at the moment.